src/test/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -1077,6 +1077,65 @@ describe("Paired Result - Load", () => {
             );
             expect(iconSVG.getAttribute("style")).toBe(`fill: ${COLORS.BLUE};`);
```

Comment:
Can we have tests for these scenarios if are not covered in this spec

- If the user sets showLine to be `true`, showShape to be `true`
- If the user sets showLine to be `false`, showShape to be `false`

Reply:
I have added spec to check if both showLine and showShape to be true. But with the second condition of having both the prop is not an ideal combination. Because the legend button will be empty if so. Do you suggest having test anyhow?

Reply:
Yeah. Add that test scenario also